<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>508</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>509</width>
    <height>508</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>509</width>
    <height>508</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>execLaunch</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_exepath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>パラメタファイルパス</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_parampath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exec">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>420</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>実行</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exepath">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>100</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_param">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_addparam">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>追加</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>どこを変えるか（１行）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_where">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_what">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>何に変えるか（カンマ区切り）</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>50</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>exeパス</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_OS">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>OS</string>
    </property>
    <widget class="QRadioButton" name="radioButton_XP">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>86</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>WinXP</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_7">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>86</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Win7</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>509</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_what</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_addparam</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_where</sender>
   <signal>returnPressed()</signal>
   <receiver>lineEdit_what</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
